hpOneView/resources/resource.py

```diff
@@ -1168,7 +1168,7 @@ def create_with_zero_body(self, uri=None, timeout=-1, custom_headers=None):
 
         return self.__do_post(uri, {}, timeout, custom_headers)
 
-    def create(self, resource, uri=None, timeout=-1, custom_headers=None, default_values={}):
+    def create(self, resource, force=False, uri=None, timeout=-1, custom_headers=None, default_values={}):
```

Comment:
add force args in the comments section as well

Reply:
Also, i would suggest force to be added in the end of args list as it would not break any existing code.